chart/chart.go

```diff
@@ -0,0 +1,214 @@
+package chart
+
+import (
+	"archive/tar"
+	"compress/gzip"
+	"errors"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/kubernetes/deployment-manager/log"
+)
+
+const ChartfileName string = "Chart.yaml"
+
+const (
+	preTemplates string = "templates/"
+	preHooks     string = "hooks/"
+	preDocs      string = "docs/"
+)
```

Comment:
No, but they will be in a subsequent commit. Maybe I'll just add those functions to this commit.

I'm also wondering if I should make them public, like `DirTemplates`.